packages/expo-modules-autolinking/src/types.ts

```diff
@@ -36,6 +46,10 @@ export interface SearchOptions {
 
   // Scratched from project's config
   flags?: Record<string, any>;
+
+  android?: {
+    buildFromSource?: string[] | null;
+  };
```

Comment:
could you share an example how to specify the buildFromSource option? feel like string[] array and nested `android.buildFromSource` is a little difficult to specify from command line

Reply:
This field should be defined inside the `package.json`, similar to the `flag` field. We can add support for passing it in `settings.gradle` later, but it isn't necessary now.

Reply:
cool thanks for sharing this information. it makes sense.